src/Redux/api.tsx

```diff
@@ -203,16 +221,21 @@ const routes = {
   partialUpdateUser: {
     path: "/api/v1/users/{username}/",
     method: "PATCH",
+    TRes: Type<{ [key: string]: object }>(),
+    TBody: Type<{ [key: string]: string | object | undefined }>(),
   },
 
   deleteUser: {
     path: "/api/v1/users",
```

Comment:
The delete user endpoint is:
<img width="461" alt="image" src="https://github.com/coronasafe/care_fe/assets/25143503/7ff11389-e3b6-475b-8e7e-033805c6d4fe">


```suggestion
    path: "/api/v1/users/{username}/",
```